components/esp32_camera.rst

```diff
@@ -192,7 +192,47 @@ Configuration for TTGO T-Camera
       # Image settings
       name: My Camera
       # ...
+      
+Configuration for ESP-Eye 
+-------------------------
+
+.. warning:: 
+
+    This camera pretty finicky:
+        - It is prone to overheat
+        - Took a couple flashes to stick
```

Comment:
```suggestion
        - Took a couple flashes to stick
```

`to stick`? Please explain what you mean by that.

Also, express this with `takes a couple ...`